app/react/Layout/DocumentsList.js

```diff
@@ -119,6 +130,20 @@ class DocumentsList extends Component {
               stateProperty={this.props.sortButtonsStateProperty}
               storeKey={this.props.storeKey}
             />
+            <div className="select-all-documents">
```

Comment:
The select all works for the other document list views, but should this button only be added on the multiEditThesaurus page?

Reply:
I think it makes sense for all.